doc/www/pocl-1.6.mak

```diff
@@ -0,0 +1,73 @@
+<%!
+        sub_page = "Portable Computing Language (pocl) v1.6 released"
+%>
+<%inherit file="basic_page.makt" />
+
+<h1>October 9, 2020: pocl v1.6 released</h2>
+
+<h2>Release Highlights</h2>
+
+<h3>Improved CUDA performance and features</h3>
+
+In pocl v1.6, the CUDA backend gained several performance improvements.
+Benchmarks using <a href="https://github.com/vetter/shoc/wiki">SHOC</a> benchmarks
+(now <a href="https://github.com/pocl/pocl/pull/847">continually tested</a>
+show that these optimizations resulted in much better performance,
+particularly for benchmarks involving local memory such as FFT and GEMM, when compared
+to a <a href="http://portablecl.org/cuda-backend.html">prior benchmark run</a>.
+Pocl now often attains performance competitive with Nvidia's
+proprietary OpenCL driver. We welcome contributions to identifying
+and removing the root causes for any remaining problem areas.
```

Comment:
Perhaps add a bit of the OpenCL 1.2/3.0 feature coverage in the CUDA implementation?